common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java

```diff
@@ -217,9 +232,14 @@ public ProduceProcessor ensureFindProduceProcessor(String acceptType) {
     if (StringUtils.isEmpty(acceptType)) {
       return defaultProcessor;
     }
-
-    List<String> mimeTyps = MimeTypesUtils.getSortedAcceptableMimeTypes(acceptType.toLowerCase(Locale.US));
-    for (String mime : mimeTyps) {
+    if (downloadFile) {
```

Comment:
Is it better to add it before StringUtils.isEmpty(acceptType)?

Reply:
ok, I will change it. actually, it doesn't matter where it's .
As   **produceProcessorMap.putIfAbsent(MediaType.WILDCARD, defaultProcessor);**
so **defaultProcessor**  equals  **produceProcessorMap.get(MediaType.WILDCARD);**

Reply:
ok